src/components/MainContent/Setting/SettingContent/SettingAppearance/SettingAppearance.js

```diff
@@ -13,6 +13,8 @@ function SettingAppearance() {
   }
 
   useEffect(() => {
+    if (displayThemeState === "") return;
+
```

Comment:
At first, we give `displayThemeState` a default "light" value
when it's first render, the display theme always sets to "light", which is not we want

we fix this by setting the default value to empty string, then add the if-statement to prevent setting the wrong display theme